import.sh

```diff
@@ -33,12 +33,12 @@ mysql -u$MYSQL_USER -p$MYSQL_PASSWORD $MYSQL_DATABASE_NAME < $DIRECTORY/structur
 mysql -u$MYSQL_USER -p$MYSQL_PASSWORD $MYSQL_DATABASE_NAME < $DIRECTORY/data.sql
 
 ## Check if we are installing Magento 1 or 2
-if [ -f "$DIRECTORY/app/etc/env.php" ]; then
-        VERSION="m2"
-        MAGERUN_COMMAND=$MAGERUN2_COMMAND
+if [ -f "$DIRECTORY/app/etc/local.xml" ]; then
+        VERSION="m1"
+        MAGERUN_COMMAND=$MAGERUN1_COMMAND
 else
-		VERSION="m1"
-		MAGERUN_COMMAND=$MAGERUN1_COMMAND
+		VERSION="m2"
+		MAGERUN_COMMAND=$MAGERUN2_COMMAND
```

Comment:
@CsSatter why this switch?

Reply:
This is because when you build a Magento Cloud environment locally it'll not provide a env.php.

Because we mostly work with Magento 2 projects I felt like it was warranted to make Magento 2 the default and check if app/etc/local.xml exists to change the project to Magento 1.

This way our tools will create a workable app/etc/env.php as well.